xml/physical_security.xml

```diff
@@ -426,4 +426,180 @@ polkit.addRule(function(action, subject) {
      </step>
     </procedure>
    </sect1>
+  <sect1 xml:id="sec-USBGuard">
+<title>System protection with <emphasis>USBGuard against rogue USB devices</emphasis></title>
+<para>
+  The USBGuard software framework helps to protect your system against rogue USB devices.
+  It does this by implementing basic whitelisting and blacklisting capabilities based on the device attributes.
+</para>
+<para>The USBGuard provides the following features:</para>
+<itemizedlist>
+  <listitem>
+   <para>
+    A command-line interface to interact with a running USBGuard instance
+    </para>
+  </listitem>
+  <listitem>
+   <para>The daemon component with an inter-process communication (IPC)
+    interface for dynamic interaction and policy enforcement</para>
+  </listitem>
+  <listitem>
+   <para>The rule language for writing USB device authorization policies</para>
+  </listitem>
+  <listitem><para>The C++ API for interacting with the daemon component implemented in a shared library</para></listitem>
+   </itemizedlist>
+   <sect2 xml:id="sec-installing-usbguard">
+    <title>Installing USBGuard</title>
+    <para>You can install and configure USBGuard.</para>
+    <procedure>
+      <step><para>To install USBGuard:</para>
+      <screen>&prompt.sudo; zypper install usbguard-tools</screen>
```

Comment:
From a technical point: `zypper install usbguard` should be enough to get a working daemon.

usbguard-tools is only needed for the CLI command that allows the user to interact with the daemon in an easier fashion, like generating policy or allowing/blocking a usb device during runtime (i.e. not adding a persitant rule to the policy that would survive reboots). usbguard-tools is not strictly needed though, one could write the rules by hand or get the rules from another host, identical host system.